README.md

```diff
@@ -111,19 +118,32 @@ aws sso login
 
 Congrats! You're all set to use `express-cli` commands.
 
+If you are using Google cloud platform, you need to configure authentication on `gcloud`.
+
+If you have downloaded the service account credentials, you can use the `GOOGLE_APPLICATION_CREDENTIALS` environment variable to provide the location of thst credential JSON file. 
+
+```bash
+gcloud auth application-default login
+
+# OR
+
+export GOOGLE_APPLICATION_CREDENTIALS='/absolute/path/to/sa/creds.json'
+```
+
 ### Commands
 
 Instructions to run `express-cli`.
 For the list of commands, please run `express-cli --help`
 First off, you need to `--init` terraform on your local machine, by executing the following command.
 
-- `./bin/express-cli --init`
+- `./bin/express-cli --init` (For Google cloud: `./bin/express-cli --init --cloud gcp`)
```

Comment:
Sure, will make the change

Reply:
We have removed `--cloud` option and used existing `--init` option. For GCP it will be `./bin/express-cli --init gcp` and for AWS `./bin/express-cli --init aws`.